### PR TITLE
Staggered orders: neutral mode and some improvements

### PR DESCRIPTION
**Pull request by bitfag**

### CODE REVIEW

File: dexbot/strategies/staggered_orders.py [hunk 1]
```diff
@@ -309,6 +315,25 @@ def log_maintenance_time(self):
...
+
+        if fee_asset['id'] == '1.3.0':
+            # Fee asset is BTS, so no further calculations are needed
+            return limit_order_create['fee']
+        else:
+            # Determine how many fee_asset is needed for core-exchange
+            temp_market = Market(base=fee_asset, quote=Asset('1.3.0'))
+            core_exchange_rate = temp_market.ticker()['core_exchange_rate']
+            return limit_order_create['fee'] * core_exchange_rate['base']['amount']
+
```
Comment (MarkoPaasila) [inline dexbot/strategies/staggered_orders.py]: Could this be in base and basestrategy?

Comment (bitfag) [reply dexbot/strategies/staggered_orders.py]: Yeah I think it's good to move to the base.py later along with migration of Staggered Orders to base.py.

Comment (joelvai) [reply dexbot/strategies/staggered_orders.py]: I will wait for that change before making this merge.

Comment (bitfag) [reply dexbot/strategies/staggered_orders.py]: @joelvai ok, changes pushed, please check.

**Pull request merged**